tests/Moq.Tests/ProtectedAsMockFixture.cs

```diff
@@ -384,12 +418,51 @@ public void VerifyGet_includes_failure_message_in_exception()
 		public interface INested
 		{
 			int Value { get; set; }
+			int Method(int value);
```

Comment:
&dagger;) Is this part of the bugfix mentioned in the changelog? If so, please re-submit as part of a separate bugfix PR.